src/stories/index.stories.js

```diff
@@ -8,6 +8,7 @@ import AccountList from '../components/AccountList.vue';
 import AccountSelector from '../components/AccountSelector.vue';
 import Address from '../components/Address.vue';
 import AddressDisplay from '../components/AddressDisplay.vue';
+import AccountAddresses from '../components/AccountAddresses.vue';
```

Comment:
What should be the name now?
Please not `AccountAddresses`....
We already have so much confusing `Account` and `Address` stuff.
Either go with `AccountIdentifier`, `AccountRing` (or `Accounticon` :stuck_out_tongue: )

Reply:
Ok, I'll take AccountRing.